CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Coding guidelines
```

Comment:
Sure, but then I'd expect this file to explain some high level things about how to contribute. Right now, the content matches a file with a different name. If we're just going to have guidelines then just call it that.